rails-snow-queen/app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,69 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ */
+$theme-colors: (
```

Comment:
Are these variables being used anywhere? 

Normally, sass variables are named using a `$` prefix. See [here](https://sass-lang.com/guide) for a starting guide.
```scss
$primary: #61D5DD
[...]
```